odinmain/src/main/kotlin/me/odinmain/features/impl/floor7/p3/TerminalTimes.kt

```diff
@@ -22,6 +27,8 @@ object TerminalTimes : Module(
         modMessage("§6Terminal PBs §fhave been reset.")
     }
 
+    private val terminalSplits: Boolean by BooleanSetting("Terminal Splits", default = true, description = "Adds the time when a term was completed to its message, and sends the total term time after terms are done.")
+
```

Comment:
Doesnt this fit way better in the splits module?

Reply:
I dont think so. It edits existing chat messages to include more info and would be rather bulky if every segment was split into a new line.